_posts/blog/2016-12-22-dwh-part1_getting_started.md

```diff
@@ -0,0 +1,134 @@
+3. DWH core: It is planned to load the cleaned and transformed data into the DWH core. The DWH core is intended to contain the entire data for general purpose analysis. This
+   contrasts the previous definitions which describe a DWH as subject-oriented. We wanted to keep that door open because analysis purposes were not clearly defined at that stage
+   and keeping the platform flexible was important to support comprehensive analysis by data scientists and data mining experts.
+4. Data mart: For our use case, the data mart is the business layer which is subject-oriented and optimized for analysis purposes. The data is loaded from the DWH core via a
+   separated ETL process and contains aggregated information which can be queried by business analysts.
+
+<center>
+<picture>
+  	<a href="/images/blog/dwh_journey/dwh_components.png"><img src="/images/blog/dwh_journey/dwh_components.png" alt="DWH components" width="850"></a>
+   	<figcaption>Figure 1: DWH components</figcaption>
+</picture>
+</center>
+
+
+## Processing steps and tools
+
+From the consultancy, we learned that a DWH can be implemented as a flexible process chain which is applied to collect data from different sources. The collected data is then
+stored in a central location and used to generate reports, create insights, and support business decision-making. We identified the following process steps: _extract, transform
+and load (ETL)_, _scheduling_, _monitoring_, _storage_, and _reporting_. As briefly described below, each of these processing steps can be covered by a variety of tools:
+
+* The ETL process is one of the most important processing steps of the DWH. It is applied to extract, transform, and load data from various sources
+  into the staging area. Available ETL tools are: Talend, Etleap, Alteryx, Pentaho, and AWS Lambda.
+* Scheduling is used to trigger specific actions time-controlled. For example, it is useful to execute long running and complex calculations overnight and not during the main
+  business hours. Another example is consecutive actions where one action waits for the successful termination of another. Some scheduling tools are: Jenkins, Talend Enterprise,
+  and SOS-Scheduler.
+* Monitoring is required to be informed about the condition of the DWH and the states of individual actions. It is crucial to be alerted automatically in case of problems.
+  Possible monitoring tools we identified are: Splunk or DataDog.
+* Storage is mandatory for a data processing where the data is persistently accommodated. Some available storage tools are: Redshift, Snowflake, PostgreSQL, and Cassandra.
+* Finally, reporting is the purpose of a DWH where, based on the data held in the DWH, business cases are summarized and visualized.
+
+## Data Lake – why would we need one?
+
+We have already gained more than 6 million monthly unique users and it seems that this number will keep growing. Thus, our data management process is clearly within the realm
```

Comment:
" 6 million monthly unique users"  - is this a number that we are allowed to make public?